src/Foundation/Site.php

```diff
@@ -166,6 +167,8 @@ protected function getApp()
         $app->register(ViewServiceProvider::class);
         $app->register(ValidationServiceProvider::class);
 
+        $app->register(BusProvider::class);
```

Comment:
No need for the `BusProvider` alias?

Reply:
Nope, we load both Illuminate BusServiceProvider and Flarum BusServiceProvider. Or we call the Illuminate from inside our own.